myriad-scheduler/src/main/java/org/apache/myriad/scheduler/NMExecutorCLGenImpl.java

```diff
@@ -164,13 +164,13 @@ public String getConfigurationUrl() {
     if (httpPolicy != null && httpPolicy.equals(TaskFactory.YARN_HTTP_POLICY_HTTPS_ONLY)) {
       String address = conf.get(TaskFactory.YARN_RESOURCEMANAGER_WEBAPP_HTTPS_ADDRESS);
       if (address == null || address.isEmpty()) {
-        address = conf.get(TaskFactory.YARN_RESOURCEMANAGER_HOSTNAME) + ":8090";
+        address = conf.get(TaskFactory.YARN_RESOURCEMANAGER_HOSTNAME) + cfg.getYarnResourceManagerPortHTTPS();
       }
       return "https://" + address + "/conf";
```

Comment:
Basically Lines 166-168 were a last ditch effort to get the HTTPS port if it didn't exist in the YarnConfiguration object, after learning more it turns out this code is only hit if yarn.resourcemanager.webapp.https.address does not exist, however it defaults to yarn.resource.manager.hostnam:8090.  So we can safely delete that code, making the additional configuration cruft unnecessary.